userspace/libsinsp/filterchecks.cpp

```diff
@@ -4442,12 +4442,15 @@ uint8_t* sinsp_filter_check_event::extract(sinsp_evt *evt, OUT uint32_t* len, bo
 							vector<string> subelements = sinsp_split(e, ':');
 							ASSERT(subelements.size() == 2);
 							m_strstorage = trim(subelements[1]);
+							if (m_strstorage.length() >= 12) {
+								m_strstorage = m_strstorage.substr(0, 12);
```

Comment:
If the length is >= 12, you truncate to 12 characters. This has an edge case where if the length is exactly 12, you will perform a meaningless truncation.